src/components/books.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+
+const Book = (props) => {
+  const { book } = props;
+  const dispatch = useDispatch();
+  return (
+    <li key={book.id}>
+      <span>{` ${book.title} `}</span>
+      <span>{`by ${book.author} `}</span>
+      <button type="button" onClick={() => dispatch(removeBook(book.id))}>Remove Book</button>
+    </li>
+  );
+};
+
+export default Book;
+
+Book.propTypes = {
+  book: PropTypes.shape({
+    id: PropTypes.number,
```

Comment:
- [OPTIONAL] Please consider handling this bug. While the app may work in development, this bug can make it not run in production. 

![error](https://user-images.githubusercontent.com/45002654/146251141-92c459bf-015a-4ebd-ae4f-e69645402ff6.png)

## The reason for this:
The id prop is expected to be a **`number`** in the component but a **`string`** is what is supplied to the component. Please refactor this to a **`string`**.

```suggestion
    id: PropTypes.string,
```